src/gui/optionsdialog.ui

```diff
@@ -837,6 +837,9 @@
             </item>
             <item>
              <widget class="QCheckBox" name="checkRecursiveDownload">
+              <property name="toolTip">
+               <string>Show recursive download confirmation if torrent contains .torrent files</string>
```

Comment:
I think it is not so clear about "recursive download confirmation" to casual users. 
My try: "When a torrent is downloaded, ask to download all .torrent files in the torrent"

@LordNyriox 
Any comment on this topic?

Reply:
> I think it is not so clear about "recursive download confirmation" to casual users. 

I am not sure if I like either version of the tooltip. 

@adem4ik's version is semi-technical (which I like), but misses a key detail (namely, that the confirmation occurs *after* the containing torrent is downloaded). 

@Chocobo1's version is a little too high-level for my own taste, but more or less effective.  I would suggest a slight wording adjustment if it is adopted: 

> When a torrent is downloaded, ask to download **any** .torrent files **within** the torrent

Reply:
@Chocobo1 & @LordNyriox I've changed the string to LordNyriox's version.